index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header class="site-header">
+        <a href="#"> Tumworobere Annah</a>
+        <nav class="nav-menu">
+            <img src="./images/Vector.png" alt="Vector image ">
+        </nav>
+   </header>
+   <main>
+    <section class="container">
```

Comment:
- [ ] Use background color for the section container same as the Figma template.
![Captura de pantalla_2021-09-21_19-27-06](https://user-images.githubusercontent.com/53111464/134264793-c3a54b12-4222-4b50-a833-6244d971f88e.png)

Reply:
Thank you @Alexoid1  for taking your time to review my project. I am going to work on the requested changes